### PR TITLE
mitmproxy 0.14.0

### PR DESCRIPTION
**Pull request by abookyun**
The latest version of mitmproxy is 0.14.0. See also: http://mitmproxy.org

### CODE REVIEW

File: Library/Formula/mitmproxy.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Mitmproxy < Formula
   desc "Intercept, modify, replay, save HTTP/S traffic"
   homepage "https://mitmproxy.org"
-  url "https://mitmproxy.org/download/mitmproxy-0.13.tar.gz"
-  sha256 "f35b90d836693dbb02a589ddee056bb4fdb7b679a1dfe230b0492216a8e3dcfa"
+  url "http://mitmproxy.org/download/mitmproxy-0.14.0.tar.gz"
```
Comment (DomT4) [inline Library/Formula/mitmproxy.rb]: Please don't downgrade `https` to `http` where the former still works, Thanks.

File: Library/Formula/mitmproxy.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  sha256 "5d74b6d8f33f5e12de16ae2d0f82f7268d73ca85"
```
Comment (DomT4) [inline Library/Formula/mitmproxy.rb]: This is a SHA1, not a SHA256. Need the latter.

**Comment by abookyun** [comment]
Corrected, @DomT4 thanks!

**Comment by DomT4** [comment]
Looks like we're missing a new resource:

```
pkg_resources.DistributionNotFound: construct>=2.5.2
Error: mitmproxy: failed
Failed executing: /usr/local/Cellar/mitmproxy/0.14.0/bin/mitmproxy --version
```

**Comment by abookyun** [comment]
Yap! just found out and patched. Let's see if it'll pass or not

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/b637592e5d16b5dc0ac0b2e7e4fef3aa2061e60a. Thank you for your contribution to Homebrew @abookyun; we appreciate it! :smiley_cat:

**Comment by abookyun** [comment]
:fireworks: :fireworks: :fireworks: @DomT4 :))

**Pull request closed without merging**